src/components/footer.js

```diff
@@ -1,19 +1,78 @@
 import React from "react"
 import PropTypes from "prop-types"
+import Section from "../styles/Section"
 import styled from "styled-components"
 import Logo from "./logo"
 import Img from "gatsby-image"
 
+const GrayBackground = styled.div`
+  background: #eaecf0;
+`
+
 const SizedLogo = styled(Logo)`
   height: 40px;
   width: 121px;
 `
 
+const Navigate = styled.div`
+  padding-right: 30%;
+  color: #2a426b;
+`
+
+const Contact = styled.div`
+  color: #2a426b;
+`
+
+const FlexSection = styled(Section)`
+  padding-top: 50px;
+  padding-bottom: 70px;
+  display: flex;
+  justify-content: space-between;
+`
+const FooterInfo = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+
+const SocialSection = styled.div`
+  display: flex;
+`
+
+const SocialInfo = ({ info }) => {
+  return (
+    <StyledSocial>
+      <a href={info.url}>
+        <Img fixed={info.icon.childImageSharp.fixed} />
+      </a>
+    </StyledSocial>
+  )
+}
+
+const StyledSocial = styled.div`
+  margin-right: 20px;
+`
+
 const Footer = ({ pages, email, facebook, linkedin }) => (
   <footer>
-    <SizedLogo />
-    <Img fixed={facebook.icon.childImageSharp.fixed} />
-    Footer
+    <GrayBackground>
+      <FlexSection>
+        <SizedLogo />
+        <FooterInfo>
+          <Navigate>
+            <h>NAVIGATE</h>
+            <p>home</p>
+          </Navigate>
+          <Contact>
+            <h>CONTACT</h>
+            <p>sandboxneu@gmail.com</p>
```

Comment:
Could we change this so when you click on the email, it opens you email .
<a href="sandboxneu@gmail.com">sandboxneu@gmail.com</a> should do.
Also @ryandrew14 @dajinchu, we can just use info@sandboxneu.com now right?

Reply:
Yes, that email address works